rali/rali_pybind/amd/rali/ops.py

```diff
@@ -396,29 +493,44 @@ class ColorTwist(Node):
     def __init__(self, brightness = 1.0, bytes_per_sample_hint = 0, contrast = 1.0, hue = 0.0, image_type = 0, 
                 preserve = False, saturation = 1.0,seed = -1, device = None):
         Node().__init__()
-        self._brightness = brightness
+        self._brightness = b.CreateFloatParameter(brightness)
         self._bytes_per_sample_hint = bytes_per_sample_hint
-        self._contrast = contrast
-        self._hue = hue
+        self._contrast = b.CreateFloatParameter(contrast)
+        self._hue = b.CreateFloatParameter(hue)
         self._image_type = image_type
         self._preserve = preserve
-        self._saturation = saturation
+        self._saturation = b.CreateFloatParameter(saturation)
         self._seed = seed
         self.output = Node()
+        self._temp_brightness = None
+        self._temp_contrast = None
+        self._temp_hue = None
+        self._temp_saturation = None
 
-    def __call__(self,input):
+    def __call__(self,input, hue = None, saturation = None, brightness = None, contrast = None):
```

Comment:
Codacy found an issue: [Redefining built-in 'input'](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5763058)

Reply:
Do we really need the statement input.next = self, it is ambiguous and will be taken care of by previous node self.next = self.output

Reply:
This is needed since every ops class has two node. self and output.
We establish a connection between nodes in define_graph().
We link input of the node (which is output of prev node) to the current node via
input.next = self.